### PR TITLE
FULL PIVOT

### PR DESCRIPTION
**Pull request by sorbae**
- Move from Scylladb to Mongodb
- Import AWS SQS consumer
- Complete inventory worker
- Complete client worker
- Complete unit and integration tests

### CODE REVIEW

File: server/inventoryWorker.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    // transpose each messages
+    // store new availabilities into db
+    done();
+  },
+});
+
+const retrieveAvailability = () => {
+// poll messages from inventories
+// check for sqs message duplicates
+  sqsConsumer.start();
```
Comment (andino93) [inline server/inventoryWorker.js]: Are you planning to start the polling process of sqs consumer elsewhere?

**Review by andino93 — APPROVED** [review]
Looks good. Couple empty functions and one thing about sqs consumer but everything else looks solid for a full pivot

**Review by ephraimg — APPROVED** [review]
Wow, big moves! New stuff looks good. Well done with the new mongo tests. When we're back I'll have to get the scoop on why you decided to switch DBs.

**Review by andino93 — APPROVED** [review]
looks good. well done on the pivot

**Pull request merged**